python/hail/api2/matrixtable.py

```diff
@@ -2460,4 +2460,41 @@ def head(self, n):
 
         return MatrixTable(self._jvds.head(n))
 
+    @handle_py4j
+    @typecheck_method(frac=numeric,
+                      seed=integral)
+    def sample_rows(self, frac, seed=1):
```

Comment:
I think the parameter name should be `p` for probability since by `sample_rows` we really do mean sample each row with probability `p` rather than get the fraction back. And a fraction doesn't always mean a number between 0 and 1. I also think `seed=0` is a more standard default. So here's my suggestion which no longer needs the note:

```
    @handle_py4j
    @typecheck_method(p=numeric,
                      seed=integral)
    def sample_rows(self, frac, seed=0):
        """Downsample the dataset by keeping each row with probability ``p``.

        Examples
        --------
        Downsample the dataset to approximately 1% of its rows.

        >>> small_dataset = dataset.sample_rows(0.01)

        Parameters
        ----------
        p : :obj:`float`
            Probability of keeping each row.
        seed : :obj:`int`
            Random seed.

        Returns
        -------
        :class:`.MatrixTable`
            Dataset with approximately ``p * num_rows`` rows.
```